### ISSUE TITLE
`TypeError: __init__() takes 1 positional argument but 2 were given` during initialisation of `pg.PlotWidget`

### ISSUE THREAD

**Issue by yconst** [pyqtgraph/pyqtgraph#2691]
### Short description
Throws `TypeError: __init__() takes 1 positional argument but 2 were given` exception during initialisation of `pg.PlotWidget` in a simple example. Same exception is thrown also in (more complex) application.

### Code to reproduce

```python
from PySide6.QtWidgets import QApplication, QMainWindow
import pyqtgraph as pg
import sys

class MainWindow(QMainWindow):

    def __init__(self):
        super(MainWindow, self).__init__()

        self.graphWidget = pg.PlotWidget()
        self.setCentralWidget(self.graphWidget)

        hour = [1,2,3,4,5,6,7,8,9,10]
        temperature = [30,32,34,32,33,31,29,32,35,45]

        # plot data: x, y values
        self.graphWidget.plot(hour, temperature)


app = QApplication(sys.argv)
w = MainWindow()
w.show()
app.exec()
```

This example was taken from here: https://www.pythonguis.com/tutorials/pyside6-plotting-pyqtgraph/

### Expected behavior
Example should run

### Real behavior
Below exception is thrown

```
Traceback (most recent call last):
  File "C:\Users\****\pyside6_test.py", line 21, in <module>
    w = MainWindow()
  File "C:\Users\****\pyside6_test.py", line 10, in __init__
    self.graphWidget = pg.PlotWidget()
  File "C:\Users\****\Anaconda3\envs\pyqtplot_test\lib\site-packages\pyqtgraph\widgets\PlotWidget.py", line 55, in __init__
    self.plotItem = PlotItem(**kargs)
  File "C:\Users\****\Anaconda3\envs\pyqtplot_test\lib\site-packages\pyqtgraph\graphicsItems\PlotItem\PlotItem.py", line 154, in __init__
    self.titleLabel = LabelItem('', size='11pt', parent=self)
  File "C:\Users\****\Anaconda3\envs\pyqtplot_test\lib\site-packages\pyqtgraph\graphicsItems\LabelItem.py", line 19, in __init__
    GraphicsWidget.__init__(self, parent)
  File "C:\Users\****\Anaconda3\envs\pyqtplot_test\lib\site-packages\pyqtgraph\graphicsItems\GraphicsWidget.py", line 18, in __init__
    QtWidgets.QGraphicsWidget.__init__(self, *args, **kwargs)
TypeError: __init__() takes 1 positional argument but 2 were given
```

### Tested environment(s)

 * PyQtGraph version: pyqtgraph-0.13.2
 * Qt Python binding: pyside6-6.5.0
 * Python version: 3.9
 * NumPy version: numpy-1.24.2
 * Operating system: Windows 10
 * Installation method: pip

### Additional context
--

**Comment by pijyoi** [comment]
Please upgrade to pyqtgraph 0.13.3

**Comment by j9ac9k** [comment]
This is a duplicate of a number of issues marked as closed as we had already had the fix merged in `master`.  We have released 0.13.3 which includes this fix too.
